localization/english/religion/wc_religion_l_english.yml

```diff
@@ -660,3 +660,27 @@
  masonry_adherent:0 "Mason"
  masonry_adherent_plural:0 "Masons"
  masonry_desc:0 "Arose among the guilds of masons, this branch of the Holy Light opposes itself to the Holy Light Church. They reject dogmas unpopular among the common folk such as the unshakable moral authority of the priests."
+ 
+ avian_group:0 "Avianism"
+ avian_group_adj:0 "Avian"
+ avian_group_adherent:0 "Avian"
+ avian_group_adherent_plural:0 "Avians"
+ avian_group_desc:0 "A group of faiths aroused among the arakkoa of Draenor consisting of four $avian_group_adherent$ gods: Rukhmar, Anzu, Sethe and the greatest King Terokk, who they worship as their fourth god."
```

Comment:
```suggestion
 avian_group_desc:0 "A group of faiths aroused among the arakkoa of Draenor consisting of four $avian_group_adherent$ gods: Rukhmar, Anzu, Sethe and Terokk, the greatest of the Arakkoan kings now worshipped as a god."
```

Reply:
Done in c25c2cd14784e04ecf0138d1a7c5f7228595160b.